app/src/main/java/com/example/moviebooking/activities/LogInActivity.kt

```diff
@@ -52,6 +53,8 @@ class LogInActivity : AppCompatActivity() {
                         editor.apply()
                     }
                     Toast.makeText(this, getString(R.string.login_successful), Toast.LENGTH_SHORT).show()
+                    val intent = Intent(this,HomeScreenActivity::class.java)
+                    startActivity(intent)
```

Comment:
you can also write it like this startActivity(Intent(this,HomeScreenActivity::class.java)) 
if you are not using that intent variable for anything else